tests/framework/ROM/tensorflow_keras/tests

```diff
@@ -4,6 +4,7 @@
     input = 'cnn1d.xml'
     csv = 'data/outCon1DClassifier.csv'
     required_libraries = 'tensorflow'
+    max_time = 1000
```

Comment:
why? explanation? Justification?

Reply:
Ha, I just asked about this in person, too. I look forward to the solution.